tests/e2e/system_test/test_full_cluster_health.py

```diff
@@ -0,0 +1,286 @@
+"""
+Test to verify cluster health/stability when it's full (75%)
+"""
+import logging
+import time
+from ocs_ci.ocs.cluster import CephCluster, get_percent_used_capacity
+from ocs_ci.ocs import constants, benchmark_operator, ocp, defaults
+from ocs_ci.ocs.perftests import PASTest
+from ocs_ci.helpers.performance_lib import run_command
+from ocs_ci.utility import templating
+from ocs_ci.helpers.helpers import get_full_test_logs_path
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.disruptive_operations import osd_node_reboot
+from ocs_ci.ocs.node import wait_for_nodes_status
+from ocs_ci.framework.pytest_customization.marks import tier3
+
+logger = logging.getLogger(__name__)
+
+
+class TestFullClusterHealth(PASTest):
+    """
+    Test Cluster health when storage is ~75%
+    """
+
+    def setup(self):
+        """
+        Setting up test parameters
+        """
+        logger.info("Starting the test setup")
+        self.percent_to_fill = 75.0
+        self.ceph_cluster = CephCluster()
+        self.nodes = None
+
+        self.benchmark_name = "FIO"
+        self.client_pod_name = "fio-client"
+
+        super(TestFullClusterHealth, self).setup()
+        # deploy the benchmark-operator
+        self.deploy_benchmark_operator()
+
+    def teardown(self):
+        """
+        The teardown of the test environment in the end.
+
+        """
+
+        self.operator.cleanup()
+        logger.info("cleanup the environment")
+        sleep_time = 5
+        time.sleep(sleep_time * 60)
+        logger.info(
+            f"Going to sleep for {sleep_time} Minute, for background cleanup to complete"
+        )
```

Comment:
In teardown we need to also handle the node failure case otherwise if this test fails the cluster goes to bad shape and the next tests in the suite gets skipped as the cluster is not healthy.
Please use restart_nodes_by_stop_and_start_teardown fixture in teardown.

Reply:
Done